src/components/FilterJobTitles.js

```diff
@@ -1,21 +1,15 @@
  import React from 'react'
 
-function FilterJobTitles() {
-  const roles = [
-    'All',
-    'Senior',
-    'Software Engineer',
-    'Frontend Engineer',
-    'Engineer',
-    'Staff Engineer',
-    'Trainer',
-    'Intern',
-  ];
+function FilterJobTitles({jobTitle}) {
+  console.log(jobTitle)
   return (
     <header className="flex bg-white p-3 gap-2 shadow-xl sticky top-0 z-10">
-      {roles.map((role) => (
-        <button key={role} className="p-2 border rounded-full px-4 hover:bg-slate-100 hover:shadow">
-          {role}
+      <button className="p-2 border rounded-full px-4 hover:bg-slate-100 hover:shadow">
+        All
+      </button>
+      {jobTitle.map((title) => (
```

Comment:
since we change `jobTitle` prop to `jobTitles`
```suggestion
      {jobTitles.map((title) => (
```